diff --git a/app/streaming/cemuhook.cpp b/app/streaming/cemuhook.cpp
index 4771faa..94a6b63 100644
--- a/app/streaming/cemuhook.cpp
+++ b/app/streaming/cemuhook.cpp
@@ -171,7 +171,7 @@ void Server::handleReceive() {
                     }
 
                     if (const char* serial = SDL_JoystickGetSerial(joystick)) {
-                        sscanf_s(serial, "%hhx-%hhx-%hhx-%hhx-%hhx-%hhx",
+                        sscanf(serial, "%16hhx-%16hhx-%16hhx-%16hhx-%16hhx-%16hhx",
                                  &response.shared.mac[0], &response.shared.mac[1], &response.shared.mac[2],
                                  &response.shared.mac[3], &response.shared.mac[4], &response.shared.mac[5]);
                     } else {
@@ -244,7 +244,7 @@ void Server::handleSend(const GamepadState& state) {
 
     SDL_Joystick* joystick = SDL_JoystickFromInstanceID(state.jsId);
     if (const char* serial = SDL_JoystickGetSerial(joystick)) {
-        sscanf_s(serial, "%hhx-%hhx-%hhx-%hhx-%hhx-%hhx",
+        sscanf(serial, "%16hhx-%16hhx-%16hhx-%16hhx-%16hhx-%16hhx",
                  &response.shared.mac[0], &response.shared.mac[1], &response.shared.mac[2],
                  &response.shared.mac[3], &response.shared.mac[4], &response.shared.mac[5]);
     } else {
diff --git a/app/streaming/vban.cpp b/app/streaming/vban.cpp
index cb7784c..625c4b5 100644
--- a/app/streaming/vban.cpp
+++ b/app/streaming/vban.cpp
@@ -105,7 +105,7 @@ bool Emitter::bind(const QHostAddress& addr, uint16_t port) {
     m_Packet.header.format_nbc = obtained.channels - 1;
     m_Packet.header.format_bit = (Header::k_DataTypeMap[obtained.format] & Header::VBAN_DATATYPE_MASK) |
                                  (Header::VBAN_CODEC_PCM & Header::VBAN_CODEC_MASK);
-    strcpy_s(m_Packet.header.streamname, STREAM_NAME);
+    strcpy(m_Packet.header.streamname, STREAM_NAME);
 
     for (uint16_t div = 1; div <= obtained.samples; ++div) {
         if (obtained.samples % div == 0 && obtained.samples / div - 1 <= 0xFF
